src/utils/sendPaginatedList.js

```diff
@@ -2,7 +2,7 @@ import { sendSuccessResponse, sendFailedResponse, sendErrorResponse } from './re
 
 async function sendPaginatedList(res, Model, filter, pageNo, limit, select, mapping) {
   try {
-    const mapKeys = Object.keys(mapping);
+    let filteredItemList, mapKeys, itemList;
```

Comment:
Why you declare here `mapKeys`, and `filteredItemList`, these keys are used only in 21-31 lines, so declare those keys inside of the block ,starting in line 21
```
if (mapping) {
  const mapKeys = Object.keys(mapping);
  const filteredItemList = await Model.find(filter).skip((pageNo - 1) * limit).limit(limit).select(select);
  ....
}
```